boltz/base.go

```diff
@@ -17,10 +17,10 @@
 package boltz
```

Comment:
I know you probably want to get this merged asap, but having these interfaces documented, at least w/ the top-level interface/struct's intent, would be nice.